InsiteTeamTask/Repositories/EventRepo/EventRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using InsiteTeamTask.MockData;
+using InsiteTeamTask.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace InsiteTeamTask.Repositories.EventRepo
+{
+    public class EventRepository : IEventRepository
+    {
+        private readonly IDataService _mockData;
+
+        public EventRepository()
+        {
+            _mockData = new MockDataService();
+        }
+
+        public Event Get(int id)
+        {
+            return _mockData.Events().Where(x => x.Id == id).FirstOrDefault();
```

Comment:
Your lambda method can be used as an argument for the `FirstOrDefault()` method, meaning the `Where()` isn't needed:
```
return _mockData.Events().FirstOrDefault(e => e.Id == id);
```